scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -1261,12 +1249,14 @@ def verify_platforms_existence(self, platform_names_to_verify, log_info=""):
         as platform_allow or integration_platforms options) is correct. If not -
         log and raise error.
         """
+        _platforms = []
         for platform in platform_names_to_verify:
             if platform in self.platform_names:
-                continue
+                _platforms.append(self.get_platform(platform).name)
```

Comment:
when `get_platform()` don't find name in aliases, then the resulting list will have None(s) instead of raising an error.

Reply:
.. unless `self.platform_names` has all platform aliases already, but why to call `get_platform()` then ?

Reply:
we need the full platform name, not the alias or shorthand